python/setup.py

```diff
@@ -225,7 +225,7 @@ def _run_cmake(self):
                 cmake_options.append('-DPYARROW_BUNDLE_ARROW_CPP=ON')
                 # ARROW-1090: work around CMake rough edges
                 if 'ARROW_HOME' in os.environ and sys.platform != 'win32':
-                    pkg_config = pjoin(os.environ['ARROW_HOME'], 'lib',
+                    pkg_config = pjoin(os.environ['ARROW_HOME'], 'lib64',
```

Comment:
This is obviously temporary - I couldn't get it to build a self-contained `.whl` without this option, is there something I'm missing wrt the build system?

Reply:
You'll have to pass `-DCMAKE_INSTALL_LIBDIR=lib` when building the C++ libraries